tfhe/src/core_crypto/gpu/algorithms/lwe_linear_algebra.rs

```diff
@@ -54,6 +54,21 @@ pub unsafe fn cuda_lwe_ciphertext_add_async<Scalar>(
         output.ciphertext_modulus(),
         rhs.ciphertext_modulus()
     );
+    assert_eq!(
+        streams.gpu_indexes[0],
+        lhs.0.d_vec.gpu_index(0),
+        "GPU error: all data should reside on the same GPU."
+    );
+    assert_eq!(
+        streams.gpu_indexes[0],
+        rhs.0.d_vec.gpu_index(0),
+        "GPU error: all data should reside on the same GPU."
+    );
+    assert_eq!(
+        streams.gpu_indexes[0],
+        output.0.d_vec.gpu_index(0),
+        "GPU error: all data should reside on the same GPU."
+    );
```

Comment:
Why not using the same kind of message than the two following asserts?

Reply:
My bad, should be the same